src/systems/force_torque/ForceTorque.cc

```diff
@@ -137,7 +137,9 @@ void ForceTorque::PostUpdate(const UpdateInfo &_info,
     for (auto &it : this->dataPtr->entitySensorMap)
     {
       // Update measurement time
-      it.second->Update(_info.simTime);
+      auto time = math::durationToSecNsec(_info.simTime);
```

Comment:
I see the same pattern is used on other sensors, but is the conversion to sec/nsec and back necessary? Can't we just call `Update(_info.simTime, false)`?

Reply:
This may be vestigial from switching over how we handled time in Fortress. I can do a follow up to fix all of the locations where we are using this pattern.

Reply:
Similar to my comment on the main thread, I think we can do away with the time logic, but we still have to cast to the base class in order to get the `Update` function that we want to use here.

Our two options would be, neither of which are particularly pretty.

```
      it.second.get()->sensors::Sensor::Update(_info.simTime, false);
      static_cast<sensors::Sensor*>(it.second.get())->Update(_info.simTime, false);
```

Reply:
I updated to my preferred call, but I'm open to suggestions.

Reply:
One other option would be to put `using Sensor::Update(..., ...)` in each of the derived classes.

Reply:
> One other option would be to put using Sensor::Update(..., ...) in each of the derived classes.

Reply:
> One other option would be to put using Sensor::Update(..., ...) in each of the derived classes.

I was just looking into that, and what's currently happening is that this function:

`public: bool Update(const std::chrono::steady_clock::duration &_now, const bool _force);`

Calls this one:

`public: virtual bool Update(const std::chrono::steady_clock::duration &_now) = 0;`

So I think this PR is correct, calling the outermost function. We should do the same for all sensors. And we should rethink whether we want the 2nd function to be public at all; it seems to me that calling it is a mistake, because it only does a subset of what the other function does.

Reply:
Follow-up PRs:

* https://github.com/ignitionrobotics/ign-gazebo/pull/1282
* https://github.com/ignitionrobotics/ign-sensors/pull/187
* https://github.com/ignitionrobotics/ign-gazebo/pull/1283